openapi.yml

```diff
@@ -29,7 +29,12 @@ paths:
     post:
       tags:
         - files
-      summary: Creates a new file resource
+      summary: |
+        Creates a new file resource.
+        Note that the 'content_type' of the file posted will be set to 'application/octet-stream' if not supplied.
+        Additional values in 'content_type' after a semicolon will be removed (e.g. ';v15' is removed in 'application/text;v15')
+        You may post JSON files with 'content_type' of 'application/x-stanford-json' to avoid 400 errors if posted JSON file is invalid.
+        The system will automatically change the content type to 'application/json' after upload.
```

Comment:
nice!  i love the text, but i'm pretty sure it belongs on the `/v1/disk/{id}` route below, in the `put` section, as that's where the actual 400 error was coming from:  https://app.honeybadger.io/projects/77112/faults/96225134

i think the `@url=` part of the faraday response is the URL to which the request was made:
```
RuntimeError: unexpected response: #<Faraday::Response:0x00007f58b55fd808 @on_complete_callbacks=[], @env=#<Faraday::Env @method=:put @request_body=#<File:/data/h2-files/jy/fi/jyfi2xh06cszo21unpzmha8tis9m> @url=#<URI::HTTPS https://sdr-api-prod.stanford.edu/v1/disk/eyJfoobarbaz> @request=#<Faraday::RequestOptions timeout=900, read_timeout=1800> @request_headers={"User-Agent"=>"Faraday v2.7.6", "Content-Type"=>"application/json", "Content-Length"=>"216381", "Authorization"=>"Bearer eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjozLCJleHAiOjE2ODY3MTM5NjB9.gKIVYRD4tDhdhWFTw6enHrdcWroREgCcR81wtwevfIA"} @ssl=#<Faraday::SSLOptions verify=true> @response=#<Faraday::Response:0x00007f58b55fd808 ...> @response_headers={"date"=>"Tue, 13 Jun 2023 03:39:23 GMT", "server"=>"Apache/2.4.41 (Ubuntu)", "cache-control"=>"no-cache", "x-request-id"=>"f374e9f7-fe04-416e-91c3-7e444f3b4c44", "x-runtime"=>"0.002156", "x-powered-by"=>"Phusion Passenger(R) 6.0.12", "strict-transport-security"=>"max-age=31536000; includeSubDomains", "status"=>"400 Bad Request", "connection"=>"close", "transfer-encoding"=>"chunked", "content-type"=>"application/json"} @status=400 @reason_phrase="Bad Request" @response_body="{\"id\":\"bad_request\",\"message\":\"Request body wasn't valid JSON.\"}">>
```

per this flow described in the README, i think things are going wrong on step 3: https://github.com/sul-dlss/sdr-api#sequence-of-operations

Reply:
I moved the openapi documentation comment to the `/v1/disk/{id}` route